lib/src/main/java/com/orange/ods/compose/component/list/OdsListItem.kt

```diff
@@ -101,9 +101,8 @@ fun OdsListItem(
         icon = icon,
         secondaryText = secondaryText,
         singleLineSecondaryText = singleLineSecondaryText,
-        overlineText = overlineText,
-        trailing = { OdsListItemTrailing(trailing = trailing) }
-    )
+        overlineText = overlineText
+    ) { OdsListItemTrailing(trailing = trailing) }
```

Comment:
Could you get back to the version with the name of the parameter:
```kotlin 
        overlineText = overlineText,
         trailing = { OdsListItemTrailing(trailing = trailing) }
     )
```
This is the last parameter and it is a composable but in this case this is not content so this is normal to specify the parameter name.